### ISSUE TITLE
The final affinity matrix

### ISSUE THREAD

**Issue by DonaldKam** [shijieS/SST#49]
As is illustrated in the paper, in training phase two affinity matrices are processd by row-wise or column-wise softmax. In test phase, how's the final affinity matrix computed since only one matrix is needed?

**Comment by shijieS** [comment]
Pls refer to [these lines](https://github.com/shijieS/SST/blob/4102bb7ee989f83d1666d756e931a8c84ab963ab/layer/sst.py#L161-L164) in my network.

```python
        x = Variable(torch.zeros(pre_num, next_num+1))
        # x[0:pre_num, 0:next_num] = torch.max(x_f[0:pre_num, 0:next_num], x_t[0:pre_num, 0:next_num])
        x[0:pre_num, 0:next_num] = (x_f[0:pre_num, 0:next_num] + x_t[0:pre_num, 0:next_num]) / 2.0
        x[:, next_num:next_num+1] = x_f[:pre_num, next_num:next_num+1]
```
